src/Libraries/CoreNodes/Web.cs

```diff
@@ -10,6 +12,16 @@ public class Web
     {
         public static string WebRequestByUrl(string url)
         {
+            // Prevent the node from executing in no network mode.
+            var session = ExecutionEvents.ActiveSession;
```

Comment:
maybe this could be a helper in dynamoServices?

Reply:
@mjkkirschner , do you mean something like :
```
public static IExecutionSession GetExecutionSession()
{
    return ExecutionEvents.ActiveSession;
}
```

Reply:
I meant something like:
`public static ThrowIfNoNetworkModeActive` - if we want to illustrate this as a pattern to other package authors.